fs/src/io_uring/file_creator.rs

```diff
@@ -69,15 +69,15 @@ impl<'a> IoUringFileCreator<'a, LargeBuffer> {
         file_complete: F,
     ) -> io::Result<Self> {
         Self::with_buffer(
-            LargeBuffer::new(buf_size),
+            PageAlignedMemory::new(buf_size).map_err(Into::<io::Error>::into)?,
```

Comment:
`PageAlignedMemory::new(buf_size)?` should work fine too - the function has known signature, so the conversion is unambiguous